docs/expanded_mission_statement.md

```diff
@@ -0,0 +1,44 @@
+AgileVentures provides software project management consulting services to charities and
+non-profits around the world.  We coordinate teams of volunteers who want to contribute
+to charitable causes and to develop their software development and project management
+skills by working in distributed online teams.
+
+The AgileVentures website provides a coordination point for new members hoping to join
+existing teams, start new projects and connect with charitable causes.  Existing teams
+generally have autonomy in how they run their projects but benefit from guidance, support
+and input from senior Agile Ventures members.
+
+Each project is autonomous in terms of choosing technologies and workflow, although Agile
+Ventures promotes a short cycle development process with prototypes being delivered to
+clients at regular intervals (e.g. weekly) for feedback that allows a team to stay on
+track to best support customer needs.
+
+We are deeply committed to open source (all code and documentation shared publicly) and
+open development (meetings and pairing recordings shared live where possible), and we
+expect all teams to be similarly committed.  We also recommend that each project maintains
+its documentation to be as terse and as up to date as possible to facilitate the
+onboarding of new members.  The nature of volunteer open source projects means a high
+turnover of participating members; ideally it will be as straightforward as possible to
+contribute value to a project.
+
+The AgileVentures web site (WSO) has been a learning project for many developers over
+the last few years, and huge amounts of work have been put in to get it where it
+currently is.  It is a testament to the dedication and commitment of everyone who has
+been involved.  However it lacks coherency since it was an experimental learning
+project where a number of competing visions have been pursued.
+
+Moving forward WSO will have a more coherent design focus; a new phase for the website
+where the emphasis is on doing few things (or perhaps even just one) exceptionally well
+rather than being an entry level project where developers implement whichever
```

Comment:
in response to comments from @sidhene and @freeranger I changed this to "entry-level project" and away from a project for "learning" or "novice" developers.